Rework Migrate section into Import and connect (#4323)
timescale/docs: _partials/_livesync-console.md

1. **Connect to your $SERVICE_LONG**

   In [$CONSOLE][portal-ops-mode], select the $SERVICE_SHORT to sync live data to. 

1. **Connect the source database and the target $SERVICE_SHORT**

   ![Postgres connector wizard](https://assets.timescale.com/docs/images/tiger-cloud-console/pg-connector-wizard-tiger-cloud.png)

   1. Click `Connectors` > `PostgreSQL`.
   1. Set the name for the new connector by clicking the pencil icon.
   1. Check the boxes for `Set wal_level to logical` and `Update your credentials`, then click `Continue`.

>    1. Enter your database credentials or a $PG connection string, then click `Connect to database`. This is the connection string for [`<pg connector username>`][livesync-tune-source-db]. $CONSOLE connects to the source database and retrieves the schema information.

Reviewer: Can you put the `This is the ...` on a newline please.

1. **Optimize the data to synchronize in hypertables**

   ![Postgres connector start](https://assets.timescale.com/docs/images/tiger-cloud-console/pg-connector-start-tiger-cloud.png)

>    1. Select the tables to sync in the `Select table` dropdown. 

Reviewer: ```suggestion
   1. In the `Select table` dropdown, select the tables to sync.
```

>    1. Click `Select table +` to add the tables you have selected for syncing.    

Reviewer: ```suggestion
   1. To select the tables to sync, click `Select table +` .    
```

>       $CONSOLE checks the table schema and, if possible, suggests the column to use as the time dimension in a hypertable.

Reviewer: $HYPERTABLE

   1. Click `Create Connector`.
   
      $CONSOLE starts $PG_CONNECTOR between the source database and the target $SERVICE_SHORT and displays the progress.

1. **Monitor synchronization**

   ![Tiger Cloud connectors overview](https://assets.timescale.com/docs/images/tiger-cloud-console/tiger-cloud-connector-overview.png)

    1. To view the amount of data replicated, click `Connectors`. The diagram in `Connector data flow` gives you an overview of the connectors you have created, their status, and how much data has been replicated.  

>     1. To review the syncing progress per table, click `Connectors` > `Source connectors`, then select the name of your connector in the table. 

Reviewer: ```suggestion
    1. To review the syncing progress for each table, click `Connectors` > `Source connectors`, then select the name of your connector in the table. 
```

1. **Manage the connector**

>    ![Edit a PG connector](https://assets.timescale.com/docs/images/tiger-cloud-console/edit-pg-connector.png)

Reviewer: ```suggestion
   ![Edit a Postgres connector](https://assets.timescale.com/docs/images/tiger-cloud-console/edit-pg-connector.png)
```

>    1. To edit the connector, click `Connectors` > `Source connectors`, then select the name of your connector in the table. Here you can rename the connector, and delete or add new tables for syncing.

Reviewer: ```suggestion
   1. To edit the connector, click `Connectors` > `Source connectors`, then select the name of your connector in the table. You can rename the connector, delete or add new tables for syncing.
```

>    1. To pause or delete the connector, click `Connectors` > `Source connectors`, then open the three-dot menu on the right and select an option. You must pause the connector before deleting it.

Reviewer: Make this into two items please.
